veg2hab/habitat.py

```diff
@@ -266,8 +266,32 @@ def try_to_determine_habkeuze(
                     kwaliteit=Kwaliteit.NVT,
                     habitatvoorstellen=true_voorstellen,
                     opmerking=f"Er zijn meerdere habitatvoorstellen die aan hun mitsen/mozaieken voldoen; zie mits/mozk_opm voor meer info in format [opgegeven vegtype, potentieel habtype, mits/mozaiek]",
-                    mits_opmerking=f"Mitsen: {[[str(voorstel.onderbouwend_vegtype), voorstel.habtype, str(voorstel.mits), str(voorstel.mits.evaluation)] for voorstel in true_voorstellen]}",
-                    mozaiek_opmerking=f"Mozaiekregels: {[[str(voorstel.onderbouwend_vegtype), voorstel.habtype, str(voorstel.mozaiek), str(voorstel.mozaiek.evaluation)] for voorstel in true_voorstellen]}",
+                    mits_opmerking="\n".join(
+                        [
+                            str(
```

Comment:
ik zou hier gewoon een fstring van maken
```
f"[{voorstel.onderbouwendvegtype}, {voorstel.habtype}, {voorstel.mits}, {voorstel.mits.evaluation}]"
```
dan is de expliciete str( ook niet meer nodig.